app/src/bcsc-theme/features/onboarding/components/PrivacyPolicyContent.tsx

```diff
@@ -1,24 +1,26 @@
 import { CardButton } from '@/bcsc-theme/components/CardButton'
-import { SECURE_APP_LEARN_MORE_URL } from '@/constants'
 import { Button, ButtonType, testIdWithKey, ThemedText, TOKENS, useServices, useTheme } from '@bifold/core'
 import { useTranslation } from 'react-i18next'
-import { Linking, ScrollView, StyleSheet, View } from 'react-native'
+import { ScrollView, StyleSheet, View } from 'react-native'
 import { SafeAreaView } from 'react-native-safe-area-context'
 
 interface PrivacyPolicyContentProps {
   onPress?: () => void
+  onLearnMore?: () => void
 }
 
 /**
  * Privacy Policy content that informs users about the app's privacy practices.
  *
  * onPress: optional function to be called when the Continue button is pressed,
  * if not provided, the Continue button will not be displayed.
+ * onLearnMore: optional function to be called when the Learn More button is pressed.
```

Comment:
question: is this actually optional?

Does the learn more button always render? If so we should make this required.